ui/src/admin/components/chronograf/AllUsersTableHeader.tsx

```diff
@@ -49,7 +49,7 @@ class AllUsersTableHeader extends Component<Props> {
               size={ComponentSize.Small}
               active={superAdminNewUsers}
               onChange={this.handleToggleClick}
-              dataTest="new-user-admins--toggle"
+              testId="new-user-admins--toggle"
```

Comment:
These changes make this PR harder to review, since they are related to a different functionality and trhey would be better addressed in a separate PR. Since no CY test is updated herein, the `testId`s could be removed, couldn't they?